qendpoint-core/src/main/java/com/the_qa_company/qendpoint/core/util/io/compress/WriteLongArrayBuffer.java

```diff
@@ -13,7 +14,7 @@
  *
  * @author Antoine Willerval
  */
-public class WriteLongArrayBuffer implements LongArray, Closeable {
+public class WriteLongArrayBuffer extends AbstractLongArray implements Closeable {
```

Comment:
Why do you want to add that to this class? From what I understood it is used by the Bitmap375, but this class isn't. Another issue is that this constructor is open, so if the user uses it to load an already loaded array, the max value of AbstractLongArray won't be loaded.

Reply:
Yeah. We don't need it here. I was just a bit overzealous in trying to make use of the abstract class. I'll revert it.